client.py

```diff
@@ -106,6 +115,8 @@ def __iter__(self):  # Await a connection
         while not self():
             yield from asyncio.sleep_ms(self._tim_short)
 
+    __await__ = __iter__
```

Comment:
This is not needed. It is only required for CPython compatibility and client.py will never run under CPython.